src/modules/AnalysisExecution/components/SubmissionCodeViewer/ResultFile/components/Summary/PopulationLevel/style.scss

```diff
@@ -26,6 +26,18 @@
   &summary-pl {
     @include summary-result-block();
     
+    border-top: 1px solid $grey;
+
+    & .#{$namespace} {
+      &table {
+        &__row {
+          &:last-child {
+            border-bottom: none;
+          }
+        }
+      }
+    }
+    
```

Comment:
Seems like absolutely the same code as above? Cannot we make shared component or move it to mixin?